SharedModules/genericProcedures.f90

```diff
@@ -4,6 +4,7 @@ module genericProcedures
 
   use numPrecision
   use openmp_func, only : ompGetMaxThreads
+  use errors_mod, only: fatalError
```

Comment:
Yes. Basically the import makes the `fatalError` available in `genericProcedures` scope and `use genericProcedures` import the entire scope including things that were imported. 
Actually this is a feature of Fortran one needs to be careful about since this is how something can sneak into a scope unknowingly.  

That being said we should `use errors_mod` directly. I just wanted to avoid horrible conflicts with other PRs.

Reply:
I see. I also think using 'use errors_mod' directly would be cleaner, even though this means changing a lot of files. Worth creating an issue I assume!